DGP_CVXPY_Code/CS_amp_optimize_BW.py

```diff
@@ -0,0 +1,61 @@
+import cvxpy as cp
+import numpy as np
+
+# Constants
+VTO = 0.5 # Threshold voltage
+KP = 50e-6   # Transconductance parameter (μnCox)
+Cox = 2.3e-3
+mu_n = KP/Cox #Mobility in m^2/Vs
+
+# Given parameters
+Rs = 10e3  # Source resistance in Ohms
+RD = 5e3  # Drain resistance in Ohms
+CL = 1e-12  # Load capacitance in Farads
+
+
+# Create optimization variables
+L = cp.Variable(pos=True)  # Width of the transistor
+VOV = cp.Variable(pos=True) # Gate-source voltage
+
+# Specification 
+ID = 500e-6
+AV = 4
+
+W = AV * L  / (RD * mu_n * VOV * Cox)
+Rout = RD
+omega_3dB = 1 / (((2/3)*cp.square(L)*AV*Rs/(RD*mu_n*VOV)) + AV*CL*VOV/(2*ID))
+
+# Calculate open-circuit time constants (OCT)

```

Comment:
Clean up your code. No old comments like this flying around. These things are confusing to anyone who looks at your code.